package.json

```diff
@@ -60,7 +60,6 @@
   },
   "scripts": {
     "devserver": "web-dev-server --root-dir docs/",
-    "preinstall": "npx npm-force-resolutions@0.0.3",
```

Comment:
no longer necessary since 11ty fixed the subdependency security issues, we had already removed the list of packages to force prior to this so this step already did nothing